linker/Mono.Linker/Driver.cs

```diff
@@ -294,6 +296,7 @@ static void Usage (string msg)
 			Console.WriteLine ("                 default is all");
 			Console.WriteLine ("   -x          Link from an XML descriptor");
 			Console.WriteLine ("   -a          Link from a list of assemblies");
+			Console.WriteLine ("   -r          Link from a list of assemblies using public roots");
```

Comment:
I am all for removing dead code but the intention of linker is to be smart and not just kill everything. Marshal.SizeOf result value is one of example how what can break badly.